addons/tmf_loadouts/loadouts/US_Rangers_2020.hpp

```diff
@@ -0,0 +1,657 @@
+	backpack[] = {"CFP_Kitbag_MCam_Grn"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector"
+	};
+	backpackItems[] = {
+		LIST_2("CUP_MAAWS_HEAT_M"),
+		"CUP_MAAWS_HEDP_M"
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backpack[] = {"B_Carryall_mcamo"};
+	secondaryWeapon[] = {"CUP_launch_Javelin"};
+	magazines[] += {"CUP_Javelin_M"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backpack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {"CUP_Javelin_M"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backpack[] = {"B_Carryall_mcamo"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ACE_Vector"};
+	backpackItems[] = {"CUP_Javelin_M"};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR };
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	backPack[] = {"CFP_Camelbak_Mule_Drab"};
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] += {"Rangefinder"};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	vest[] = {"CUP_V_CPC_light_mc"};
+	backpack[] = {"B_Carryall_mcamo"};
+	hmd[] = {"CUP_NVG_GPNVG_black"};
+	silencer[] = {"cup_muzzle_snds_m16_coyote"};
+	sidearmWeapon[] = {"CUP_hgun_Glock17_blk"};
+	sidearmAttachments[] = {
+		"optic_mrd_black",
+		"hlc_acc_tlr1"
+	};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		"SmokeShell",
+		"SmokeShellGreen",
+		LIST_3("CUP_17Rnd_9x19_glock17")
+	};
+	backpackItems[] = {
+		LIST_4("CUP_20Rnd_762x51_B_M110"),
+		"CUP_srifle_M110_ANPVS10",
+		"cup_muzzle_snds_m110"
+	};
+	linkedItems[] += {"ItemGPS"};
+};
+
+class sp : r {
+	displayName = "Spotter";
+	backpack[] = {"CFP_Kitbag_MCam_Grn"};
+	silencer[] = {"cup_muzzle_snds_m16_coyote"};
+	items[] += {"ACE_MapTools"};
+	backpackItems[] = {
+		LIST_4("CUP_20Rnd_762x51_B_M110"),
+		"ACE_SpottingScope",
+		"ACRE_VHF30108SPIKE"
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"CUP_SOFLAM",
+		"B_UavTerminal"

```

Comment:
UAV terminal?

Reply:
To simulate the information network integration that F-35 and F-22 pilots have with other ISR assets within the area. Doubles as a GPS.